lib/endpoint/html-to-json.js

```diff
@@ -35,6 +35,10 @@ async function htmlToJson (html) {
       .get()
   }
 
+  if (gheVersion) {
+    state.gheVersion = `ghe-${gheVersion}`
```

Comment:
I was also thinking that this could live on `result.gheVersion`, i.e. it would be included in the final output? Decided not to do it just yet as it results in an update to all the routes for GHE, but it's an easy fix - let me know if you think it's worth it.